cg/cli/demultiplex/finish.py

```diff
@@ -42,13 +41,10 @@ def finish_all_cmd(context: CGConfig, bcl_converter: str, dry_run: bool) -> None
 
 @finish_group.command(name="flow-cell")
 @click.argument("flow-cell-name")
-@OPTION_BCL_CONVERTER
```

Comment:
I'm not 100 % sure we can remove this one if we need to be able to override the conversion software based on the machine type

Reply:
true, I will leave this on hold for now

Reply:
fixed